### ISSUE TITLE
Version 5.0.0 breaks Dropwizard functional tests by including dependency on org.slf4j:slf4j-api:2.0.0-alpha1

### ISSUE THREAD

**Issue by aharin** [gocardless/gocardless-pro-java#79]
After upgrading to v 5.0.0 functional tests for our dropwizard application started failing.

This seems to be due to transitive dependency on org.slf4j:slf4j-api:2.0.0-alpha1

```
java.lang.IllegalStateException: Unable to acquire the logger context
	at io.dropwizard.logging.LoggingUtil.getLoggerContext(LoggingUtil.java:46)
	at io.dropwizard.logging.DefaultLoggingFactory.<init>(DefaultLoggingFactory.java:78)
	at io.dropwizard.Configuration.getLoggingFactory(Configuration.java:107)
	at io.dropwizard.testing.DropwizardTestSupport.stopIfRequired(DropwizardTestSupport.java:267)
	at io.dropwizard.testing.DropwizardTestSupport.after(DropwizardTestSupport.java:239)
	at io.dropwizard.testing.DropwizardTestSupport.before(DropwizardTestSupport.java:231)
```

The current production version of org.slf4j:slf4j-api is 1.7.32. Would it be possible to use this version instead?

**Comment by vdandugc** [comment]
thanks for reporting, we will check this

**Comment by vdandugc** [comment]
this should be fixed by [5.0.1](https://repo.maven.apache.org/maven2/com/gocardless/gocardless-pro/5.0.1/)
